### PR TITLE
python3.pkgs.sphinxcontrib-log-cabinet: init at 1.0.1

### PR DESCRIPTION
**Pull request by KAction**
- python3.pkgs.sphinx-pallets-theme: init at 2.0.2
- python3.pkgs.sphinxcontrib-log-cabinent: init at 1.0.1

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by pbsds** [comment]
This pr seems to also include the changes in #172731.
I suggest you either rebase this pr to not include the other PR, or to close the other PR.

Result of `nixpkgs-review pr 172735` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python310Packages.sphinx-pallets-theme</li>
    <li>python310Packages.sphinxcontrib-log-cabinet</li>
    <li>python39Packages.sphinx-pallets-theme</li>
    <li>python39Packages.sphinxcontrib-log-cabinet</li>
  </ul>
</details>

**Comment by flokli** [comment]
Same review comments from https://github.com/NixOS/nixpkgs/pull/172731 apply here, too:

 - please fetch from pypi rather than github
 - please rebase this to only PR in sphinxcontrib-log-cabinet, not the other PR. They are unrelated packages.

**Comment by pbsds** [comment]
Seems like the other or got merged. I suggest we close this one

**Comment by KAction** [comment]
No, we need log-cabinet. I dropped pallets-sphinx-themes commit.

File: pkgs/development/python-modules/sphinxcontrib-log-cabinet/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  src = fetchFromGitHub {
+    owner = "davidism";
+    repo = "sphinxcontrib-log-cabinet";
+    rev = version;
+    sha256 = "03cxspgqsap9q74sqkdx6r6b4gs4hq6dpvx4j58hm50yfhs06wn1";
+  };
+
+  propagatedBuildInputs = [ sphinx ];
+
+  pythonImportsCheck = [ "sphinxcontrib.log_cabinet" ];
```
Comment (mweinelt) [inline pkgs/development/python-modules/sphinxcontrib-log-cabinet/default.nix:16]:
```suggestion
  pythonImportsCheck = [ "sphinxcontrib.log_cabinet" ];
  
  doCheck = false; # no tests
```

File: pkgs/development/python-modules/sphinxcontrib-log-cabinet/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchFromGitHub, sphinx }:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-log-cabinet";
+  version = "1.0.1";
```
Comment (mweinelt) [inline pkgs/development/python-modules/sphinxcontrib-log-cabinet/default.nix:5]:
```suggestion
  version = "1.0.1";
  format = "setuptools";
```

File: pkgs/development/python-modules/sphinxcontrib-log-cabinet/default.nix [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+  src = fetchFromGitHub {
+    owner = "davidism";
+    repo = "sphinxcontrib-log-cabinet";
+    rev = version;
```
Comment (mweinelt) [inline pkgs/development/python-modules/sphinxcontrib-log-cabinet/default.nix:10]:
```suggestion
    rev = "refs/tags/${version}";
```

**Pull request closed without merging**